client/components/ProductOverview/ImageGallery.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { AiOutlineArrowLeft, AiOutlineArrowRight } from 'react-icons/ai';
+import PropTypes from 'prop-types';
+
+const ImageGallery = ({ currentStyle }) => {
+  const [currentImageIndex, changeImage] = useState(0);
```

Comment:
For naming here, you'll want it to be something like `[currentImageIndex, setCurrentImageIndex]`.

Reply:
refactored.